words/words.md

```diff
@@ -332,21 +332,21 @@ Ora, giochiamo di nuovo con il modello SEIR, ma mostriamo R<sub>0</sub>, R nel t
 
 <!--This is because when there are more <span class="nowrap">non-<icon s></icon>s</span> than the herd immunity threshold, you get R < 1. And when R < 1, new cases stop growing: a peak.-->
 
-Questo perché quando ci sono i <span class="nowrap">non-<icon s></icon>s</span> in numero maggiore della soglia di immunità di gregge, hai che R < 1. E quando R < 1, i nuovi casi smettono di crescere: un picco.
+Questo perché quando i <span class="nowrap">non-<icon s></icon>s</span> superano la soglia di immunità di gregge, R < 1. E quando R < 1, i nuovi casi smettono di crescere: un picco.
 
 <!--**If there's only one lesson you take away from this guide, here it is** – it's an extremely complex diagram so please take time to fully absorb it:-->
 
-**Se c'è una sola lezione che ti porti a casa da questa guide, eccola** – è un diagramma estremamente complesso quindi per favore prenditi il tempo che ti serve per capirlo per bene:
+**Se c'è una sola lezione che da portare a casa da questa guida, eccola** – è un diagramma estremamente complesso, quindi prenditi pure il tempo che ti serve per metabolizzarlo per bene:
 
 ![](pics/r3.png)
 
 <!--**This means: we do NOT need to catch all transmissions, or even nearly all transmissions, to stop COVID-19!**-->
 
-**Ciò significa: NON dobbiamo impedire tutte i contagi, o neanche quasi tutti i contagi, per fermare il COVID-19!**
+**Questo significa: NON è necessario intercettare tutti i contagi, e neanche "quasi tutti" i contagi, per fermare il COVID-19!**
 
 <!--It's a paradox. COVID-19 is extremely contagious, yet to contain it, we "only" need to stop more than 60% of infections. 60%?! If that was a school grade, that's a D-. But if R<sub>0</sub> = 2.5, cutting that by 61% gives us R = 0.975, which is R < 1, virus is contained! (exact formula:[^exact_formula])-->
 
-E' un paradosso. Il COVID-19 è estremamente contagioso, ma per fermarlo, dobbiamo "solo" impedire più del 60% delle infezioni. 60%?! Se fossimo a scuola sarebbe al più una sufficienza scarsa. Ma se R<sub>0</sub> è 2,5, tagliarlo del 61% ci dà R = 0,975, per cui R < 1, e il virus è fermato! (formula esatta:[^exact_formula])
+E' un paradosso. Il COVID-19 è estremamente contagioso, ma per fermarlo ci basta che la percentuale di contagi che impediamo sia superiore "solo" al 60%. 60%?! Se fossimo a scuola sarebbe al più una sufficienza scarsa. Ma se R<sub>0</sub> è 2,5, tagliarlo del 61% ci dà R = 0,975, per cui R < 1, e il virus è contenuto! (formula esatta:[^exact_formula])
```

Comment:
ok for changing "fermato" -> "contenuto", then let's do it also at the beginning ("ma per fermarlo" -> "ma per contenerlo"). Regarding the new tranlsation of "we "only" need to stop more than 60% of infections" I personally prefer the original translation, maybe changing "infezioni" -> "contagi". This is such an important statement and with this new proposed translation it really gets convoluted. I would also change "più del" to "almeno il". The final version would be:

"dobbiamo "solo" impedire almeno il 60% dei contagi".

I think that, even out of context, it is pretty understable and memorable statement!

Reply:
I do not agree, but I do not disagree. Changed.